FlowCrypt/Controllers/Settings/Backup/Backups Seleckt Key Scene/BackupSelectKeyViewController.swift

```diff
@@ -85,14 +88,14 @@ extension BackupSelectKeyViewController {
             .filter { $0.1 == true }
             .map(\.0)
 
-        backupService.backupToInbox(keys: backupsToSave, for: userId)
-            .then(on: .main) { [weak self] in
+        cancellable = backupService.backupToInbox(keys: backupsToSave, for: userId)
```

Comment:
why not to store it in cancelable set instead of assigning it?
```
backupService.backupToInbox(keys: backupsToSave, for: userId)
            .receive(on: DispatchQueue.main)
            .sinkFuture(receiveValue: { [weak self] in
                self?.hideSpinner()
                self?.navigationController?.popToRootViewController(animated: true)
            }, receiveError: { [weak self] error in
                self?.handleCommon(error: error)
            })
            .store(in: &cancellable)
```

Reply:
I guess this two are equivalent, or?

Reply:
actually the same, I was just wondering why you decide to assign it instead of chaining in `.store`

Reply:
I prefer to assign